roles/engine/templates/testnet/settings.toml.j2

```diff
@@ -0,0 +1,60 @@
+[btc.rpc]
+basic_auth_user = "flip"
+basic_auth_password = "flip"
+http_endpoint = "{{ vault[env].engine.btc_http_rpc_endpoint }}"
```

Comment:
this file is in testet directory, so env isnt really needed, we can just use tetsnet